modules/volume/rawvolumereader.inl

```diff
@@ -103,7 +105,22 @@ std::unique_ptr<RawVolume<VoxelType>> RawVolumeReader<VoxelType>::read() {
         throw ghoul::RuntimeError("Error reading volume file");
     }
 
-    return volume;
+    if (invertZ) {
+        for (int i = 0; i < volume->nCells(); ++i) {
```

Comment:
Can move the definition of `newVolume` just after the `if` statement line and then return at the end of this if block